kratos/includes/model_part.h

```diff
@@ -108,7 +108,7 @@ class KRATOS_API(KRATOS_CORE) ModelPart : public DataValueContainer, public Flag
     typedef Dof<double> DofType;
     typedef std::vector< DofType::Pointer > DofsVectorType;
     typedef Kratos::Variable<double> DoubleVariableType;
-    typedef Kratos::VariableComponent<Kratos::VectorComponentAdaptor<Kratos::array_1d<double, 3>>> VariableComponentType;
+    typedef Kratos::Variable<double> VariableComponentType;
```

Comment:
I would say we remove this `VariableComponentType` all together ... !

Reply:
The problem is that this is used in many places, [even by you](https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/processes/apply_periodic_boundary_condition_process.h#L40) or [here](https://github.com/KratosMultiphysics/Kratos/blob/master/applications/CableNetApplication/custom_processes/sliding_edge_process.h#L49)

But maybe we can change those occurances instead, i.e. from `ModelPart::VariableComponentType` to `ModelPart::DoubleVariableType` 
=> this would be search-replace

what do you think?

Reply:
ok it is not really used anywhere, then lets remove it completely